components/LocationCard/index.js

```diff
@@ -64,6 +101,26 @@ const ReturnLink = styled(Link)`
 export default function LocationCard({ location }) {
   const router = useRouter();
 
+  const [locations, setLocations] = useLocalStorageState("favoriteLocations", {
+    defaultValue: [],
+  });
+
+  const handleEdit = () => {
+    router.push(`/EditForm/${location.id}`);
+  };
```

Comment:
You pass the id down to this function, but you don't use it. Why not?
How about this:
```suggestion
  const handleEdit = (id) => {
    router.push(`/EditForm/${id}`);
  };
```
By relying _only on the input of the function_ (the parameters) you keep your functions _pure_. And we want pure functions to avoid unexpected side effects.